### PR TITLE
Feat/doc and test traced

### PR DESCRIPTION
**Pull request by loicmathieu**
Add `@Traced` to the documentation. It exists in the quickstart so it's worth mentioning it inside the guide.

Update the `main` integration test to use `@Traced` in another bean as the REST endpoint is traced by default and it reflect the typical usage of `@Traced`.

### CODE REVIEW

File: docs/src/main/asciidoc/opentracing.adoc [hunk 1]
```diff
@@ -165,6 +165,33 @@ Then visit the http://localhost:16686[Jaeger UI] to see the tracing information.
...
+@ApplicationScoped
+public class FrancophoneService {
+
+    public String bonjour() {
+        return "bonjour";
+    }
+}
+----
+
+NOTE: this is the best way to add OpenTracing capability to reactive messaging based application by adding the `Traced` annotation to all incoming methods.
```
Comment (gastaldi) [inline docs/src/main/asciidoc/opentracing.adoc:193]:
```suggestion
NOTE: the best way to add OpenTracing capability to reactive messaging based applications is by adding the `Traced` annotation to all incoming methods.
```

**Review by gastaldi — APPROVED** [review]
LGTM, but it would be nice to squash before merging these commits :+1:

**Comment by loicmathieu** [comment]
@gastaldi I applied your suggestions

**Comment by loicmathieu** [comment]
@gastaldi I keep two commits as they are unrelated changes: documentation and test enhancement

**Comment by gastaldi** [comment]
@loicmathieu cool, maybe the commit with my suggestion can be squashed into the first commit then :wink:

**Comment by loicmathieu** [comment]
> @loicmathieu cool, maybe the commit with my suggestion can be squashed into the first commit then 😉

Already done, I always squashed suggestion commits a few seconds/minutes after commiting them, you was too hurry ;)

**Comment by gastaldi** [comment]
Thanks!

**Pull request merged**